applicationinsights.ts

```diff
@@ -133,6 +168,35 @@ export function start() {
     return Configuration;
 }
 
+export function setRetry(setUseDiskRetryCaching: boolean, setResendInterval: number, setMaxBytesOnDisk: number) {
```

Comment:
Why adding this method? setUseDiskRetryCaching should be able to have this code, not seeing this used anywhere else